01-ActivityModel/activity-model/activity_model/__main__.py

```diff
@@ -0,0 +1,28 @@
+import yaml
+
+from data_preparation import Epc, Spenser
+from enriching_population import EnrichingPopulation
+
+if __name__ == "__main__":
+    print("hi")
```

Comment:
I have add some suggestion in the #9

Reply:
@patricia-ternes @nickmalleson I have tested successfully the lastest code in this branch. I was able to run it for Leeds, and I have just a few comments that I have included in #9. Here are again my comments:

1. I think the readme can also include some instructions on how o run the model., more like guide the user to going from one readme to another one. Or maybe the model can be wrapped in another folder structure.
2. In the readme of /config, is there any way to run it to the whole local authorities?
3. Once the required parameters are set, the rest of the text can be part of some section like` extended or expand the model to the users needs.`
4. I replaced the hi print for another message.
5. We could create some kind of script to help users to know the lad code the program need to get the new LAD area. I can provide some example in a next PR.